contributing.md

```diff
@@ -10,22 +27,43 @@ You can check out [here](https://www.learncpp.com/) to get your started about C+
 
 Regarding Pybind, Here is a good [introduction](https://www.youtube.com/watch?v=jQedHfF1Jfw) about it.
 
-## Setting up your Environment
 
-### Install Script
+## Setup
+
+### Forking a Repository
+
+To contribute to PyDP you will need to fork the OpenMind/PyDP repository.
+Then you can work risk-free on your fork.
+
+You will just need to fork once. After that you can call `git fetch upstream` and `git pull 'branch-name'` before you do your local changes to get the remote changes and be up-to-date
```

Comment:
Maybe just call `git pull`?
Makes the process simpler for everyone

Reply:
okey

Reply:
specifying the branch name - as we do all our changes on the dev branch right! For local git usage, this might be good to know for beginners. Or shall I mention the branch name as well to remove any room for errors!

 Anyways, if desktop version is used then this won't be much of a concern! As :dev is set to default

Reply:
Yes, please mention it as dev for all contribution purpose.